cmd/root.go

```diff
@@ -39,6 +41,20 @@ func mustFlagString(cmd *cobra.Command, name string, required bool) string {
 	return val
 }
 
+type EnrollTokenData struct {
+	Token    string `json:"token"`
+	ServerID string `json:"serverId"`
+}
+
+func initConfig() {
+	viper.SetConfigName("config")
+	viper.AddConfigPath("./dataDir")
```

Comment:
you cant use `/` in path names this breaks on windows. also you're hard coding this here but allowing it to be passed in elswhere which will conflict